src/renderer/coremods/settings/pages/Addons.css

```diff
@@ -46,3 +46,12 @@
 .replugged-addon-breadcrumbsInactive:hover {
   color: var(--header-primary);
 }
+
+.rp-addon-header-buttons {
+  align-self: normal;
```

Comment:
You don't need this if you remove `align={Flex.Align.START}` from the parent `Flex`. The align START is no longer necessary, can be removed, style does not change with or without an open addon settings.